platform/src/apis/EducationBenchmarking.Platform.Api.Establishment/Search/LocalAuthoritySearchService.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EducationBenchmarking.Platform.Infrastructure.Search;
+using EducationBenchmarking.Platform.Shared;
+using Microsoft.Extensions.Options;
+
+namespace EducationBenchmarking.Platform.Api.Establishment.Search;
+
+public class LocalAuthoritySearchServiceOptions : SearchServiceOptions
+{
+    public string LocalAuthorityIndexName { get; set; }
+}
+
+public class LocalAuthoritySearchService : SearchService, ISearchService<LocalAuthority>
+{
+    private static readonly string[] Facets = { ""};
+    
+    public LocalAuthoritySearchService(IOptions<LocalAuthoritySearchServiceOptions> options) : base(options.Value.Endpoint, options.Value.LocalAuthorityIndexName, options.Value.Credential)
+    {
+    }
+
+    public async Task<SearchOutput<LocalAuthority>> SearchAsync(PostSearchRequest request)
+    {
+        return await SearchAsync<LocalAuthority>(request, CreateFilterExpression, Facets);
+    }
+
+    private string CreateFilterExpression(List<SearchFilters> requestFilters)
```

Comment:
Just a general question, but is this is a placeholder for now, and would we expect to fill it out?

Reply:
Yes - however it isn't a placeholder, it's intentional as there isn't any knew filters for local authorities. By put this pattern in place it will allow you to see where to add the implementation as and when we know more.